lighthouse-core/report/report-generator.js

```diff
@@ -261,14 +262,16 @@ class ReportGenerator {
     });
 
     const template = Handlebars.compile(this.getReportTemplate());
+    reportContext = reportContext || 'extension'; // could be: devtools, extension
```

Comment:
k. added more detail to jsDoc and moved the default value to top of function. this work for you? 

I could also rename to reportStyle if you prefer, works for me.